lib/cosmos/interfaces/protocols/template_stream_protocol.rb

```diff
@@ -141,7 +143,7 @@ def read_packet(packet)
 
         return result_packet
       else
-        return :STOP
+        return packet
```

Comment:
It was useful in the spec and I thought it would be good to not just eat those responses but generate UNKNOWN packets.